developer-docs-site/docs/standards/digital-asset.md

```diff
@@ -86,7 +85,7 @@ See [Aptos Token](#aptos-token) for examples on how Royalty's `MutatorRef` can b
 Royalty can also be set directly on a token if it has a different royalty config than the collection's.
 
 ## Token lifecycle
-All token v2 modules are deployed at `0x4`.
+All Digital Asset modules are deployed at `0x4`.
```

Comment:
```suggestion
All Digital Asset modules are deployed at the address `0x4`.
```

Maybe this? Or "reserved framework address" or something. Just to be a little clearer, some people might not know we have framework modules here and `0x4` doesn't look like an address to someone new

Reply:
Good point. Let's go with "reserved framework address".